web/src/components/EventsDateFilter.tsx

```diff
@@ -66,17 +29,24 @@ const EventsDateFilter = ({
   setEndDate,
 }: EventsDateFilterProps): ReactElement => {
   const [showDateFilter, setShowDateFilter] = useState(false)
+  const { mobile } = useDimensions()
   const { t } = useTranslation('events')
 
   const today = DateTime.now()
   const inAWeek = DateTime.now().plus({ week: 1 })
+  const formattedStartDate = startDate?.toFormat('dd.MM.yyyy') ?? '∞'
+  const formattedEndDate = endDate?.toFormat('dd.MM.yyyy') ?? '∞'
 
   return (
     <>
-      <StyledAccordion disableGutters expanded={showDateFilter} elevation={0}>
-        <FilterToggle isDateFilterActive={showDateFilter} setToggleDateFilter={setShowDateFilter} />
+      <Accordion disableGutters expanded={showDateFilter} elevation={0}>
+        <AccordionSummary onClick={() => setShowDateFilter(!showDateFilter)} disableRipple>
```

Comment:
there is a recommendation from WAI to set those attributes
```suggestion
        <AccordionSummary onClick={() => setShowDateFilter(!showDateFilter)} disableRipple id="panel-header" aria-controls="panel-content">
```